spec/Section 4 -- Composition.md

```diff
@@ -14,6 +14,58 @@ run in sequence to produce the composite execution schema.
 
 ### Pre Merge Validation
 
+#### Enum Type Values Must Be The Same Across Source Schemas
+
+**Error Code**
+
+ENUM_VALUES_MUST_BE_THE_SAME_ACROSS_SCHEMAS
+
+**Formal Specification**
+
+- Let {enumsByName} be a map where the key is the name of an enum type, and the value is a list of all enum types from different source schemas with that name.
+- For each {listOfEnum} in {enumsByName}:
+  - {EnumAreMergeable(listOfEnum)} must be true.
+
+EnumAreMergeable(enums):
+
+- Let {values} be the set of all values of the first enum in {enums}
+- For each {enum} in {enums}
+  - Let {enumValues} be the set of all values of {enum}
+  - {values} must be equal to {enumValues}
+
+**Explanatory Text**
+
+This rule ensures that enum types with the same name across different source schemas in a composite schema have identical sets of values. 
+Enums, must be consistent across source schemas to avoid conflicts and ambiguities in the composite schema.
```

Comment:
At the moment enum value have to be strictly equal. We do not differentiate between output, input or combined enums

Reply:
> At the moment enum value have to be strictly equal

Do we plan to explore relaxing this to allow evolution of enums?

Reply:
Yes we just talked about this in yesterdays wg, we are probably going to use the same pattern that federation uses currently.
union in ouput, intersection in input and exact match if used as in- and output